src/media/mod.rs

```diff
@@ -23,6 +23,9 @@ pub(crate) use inner::{MediaInner, Source};
 /// Half internal structures regarding RTP level.
 pub mod rtp {
     pub use crate::rtp::{Extension, ExtensionMap, ExtensionValues};
+
+    // Exposed for integration tests, not for general usage.
+    pub use crate::rtp::RtpHeader;
```

Comment:
I think you should expose utilities for rtp packet creation and parsing etc. Because even that people want to forward rtp packets they still might want to repacketize/serialize in some other way. So having the utilities available for this would be good.

Reply:
Isn't that inviting people to the dark side? 😏

Reply:
I mean, people are gonna do it anyway. Either they write their own header or take it from another crate :)

Reply:
The question is where to stop. I mean I can expose `RtpHeader`, but what about packetizer/depacketizer?

Feels like potentially a whole other crate. rtp-tools or somethin.

Reply:
I see, if people want to packetize h264 for themself? 

Tbh, unless it can break things hard I don't see it as a bad thing. I would also let people test out their own packetizers easily, expand on the current ones.

But yes, once you make something public it's hard to go back.

Up to you, worst case people will just copy the code out and have a fork of it.

Reply:
For now I'd like str0m to stay narrowly focused on what it does do rather than exposing a whole new range of API. 

People that reach for the rtp-mode API are aiming for a use case that while supported, but isn't the best experience of using str0m. It's advanced usage and you're a bit on your own. I'll make updates to the doc to this effect.